apparmor.d/groups/freedesktop/xdg-open

```diff
@@ -10,6 +10,7 @@ include <tunables/global>
 profile xdg-open @{exec_path} flags=(attach_disconnected) {
   include <abstractions/base>
   include <abstractions/consoles>
+  include <abstractions/consoles>
```

Comment:
Remove it, it is already there.

Reply:
I used a find and replace for all `/dev/tty` to consoles. I guess I missed a duplicate, will look for more.